src/injected-js/gmail/setup-gmail-interceptor.js

```diff
@@ -322,11 +322,13 @@ export function setupGmailInterceptorOnFrames(
             }
 
             try {
-              const responseParsed = parseComposeResponseBody(
+              const responsesParsed = parseComposeResponseBody(
                 connection.originalResponseText
               );
 
-              if (responseParsed) {
+              console.log('responsesParsed', responsesParsed);
+
+              for (const responseParsed of responsesParsed) {
```

Comment:
since response can contain data for different draftIds, we should not iterate through array, because basically then triggerEvent will be fired for incorrect draftId that is in this context.

not saying it should be implemented like this, but essentially what we looking for is eaither to pass DraftId into `parseComposeResponseBody` and expect result to be relevant to passed DraftId, or filter result `const responseParsed = responsesParsed.filter(r => r.draftId === draftId)`

Reply:
See https://github.com/InboxSDK/InboxSDK/pull/824/files#diff-f1974847c9b840296a92177ba6e677644336edbbf2656c52118fb96a9b51a6a1R331 for my stab at the latter approach.